tests/unit/store/holochain.spec.hc.js

```diff
@@ -47,6 +47,7 @@ describe('holochain store in holochain env', () => {
     expect(isHoloHosted()).toBe(false)
   })
 
+  // tech-debt: update test to follow approach that is not based on timing / system dependant
```

Comment:
```suggestion
  // tech-debt: update test to follow approach that is not based on timing / system dependent
```

Reply:
Haha! I just did the same update locally! Good eye. :D